arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -1068,15 +1360,15 @@
         },
         "usage": {
           "type": "string",
-          "description": "Gets or sets usage"
+          "description": "Gets or sets usage. Acceptable values are 'Verify' and 'Sign'."
```

Comment:
Remove "Gets or sets" from everywhere. AutoRest will automatically add it as per its immutability.

Secondly, didn't we decide this to be an enum with one value "Verify"